src/app/route/index.js

```diff
@@ -261,11 +322,14 @@ class AppRouter extends PureComponent {
 
     render() {
         return (
-            <Router history={ history }>
-                { this.renderRouterContent() }
-            </Router>
+            <>
+                <Meta />
+                <Router history={ history }>
+                    { this.renderRouterContent() }
+                </Router>
+            </>
         );
     }
 }
 
-export default AppRouter;
+export default connect(mapStateToProps, mapDispatchToProps)(AppRouter);
```

Comment:
Maybe it is time to remove router from index.js? I think it is a bad place for it in the end of the day.

Reply:
That is FYI no action required.

Reply:
Yes!